extensions/neo4j/deployment/src/main/java/io/quarkus/neo4j/deployment/Neo4jBuildTimeConfig.java

```diff
@@ -0,0 +1,14 @@
+package io.quarkus.neo4j.deployment;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(name = "neo4j", phase = ConfigPhase.BUILD_TIME)
+public class Neo4jBuildTimeConfig {
+    /**
+     * Whether or not an healt check is published in case the smallrye-health extension is present (default to true).
```

Comment:
Maybe we can remove the `(default to true)`  from the description. It will be automatically picked from ConfigItem during doc generation.

Reply:
Yes but the JavaDoc is also usefull for Quarkus developpers :)
Maybe I could but best is to make it after to keep all health check enabled comments the same (agroal, these 3 new ones, kafka-streams)